vendor/bat-native-ledger/BUILD.gn

```diff
@@ -168,14 +168,19 @@ source_set("ledger") {
     "//third_party/re2",
     "//url",
     rebase_path("bat-native-ads", dep_base),
-    rebase_path("bat-native-confirmations", dep_base),
     rebase_path("bat-native-anonize:anonize2", dep_base),
     rebase_path("bip39wally-core-native:bip39wally-core", dep_base),
     rebase_path("bat-native-tweetnacl:tweetnacl", dep_base),
     rebase_path("bat-native-rapidjson", dep_base),
     rebase_path("brave_base", dep_base),
   ]
 
+  if (brave_ads_enabled) {
```

Comment:
@tmancey is this needed?

Reply:
@NejcZdovc this is required, thanks

Reply:
so it's needed on android, but was not needed on desktop?

Reply:
would be good to have on desktop too

Reply:
ok

Reply:
 @tmancey @NejcZdovc that is on of problems why iOS CI fails, it doesn't have brave_ads_enabled flag turned on for some reason, for now we just removed it. You can fix that in follow ups.

Reply:
we can't merge until CI passes, so I guess we should revert this change or fix it on iOS side so that it passes

Reply:
Fixed in iOS